### DISCUSSION TITLE
Use of the jumperpads

### DISCUSSION THREAD

**Discussion by ichingxs4all** [frntc/SIDKick-pico#15] [Q&A]
I want to use an alternative PCM5102 DAC board. This board is powered by at least 5V.
It has it's own 1117 3.3v powerregulator.

Can you use the jumperpads on the SidKick board to switch the DAC power from 3.3v to 5 V. I think it is default set to 3.3v and probably have to cut it and solder it to the 5v side. 
Is this the way to do it ?

**Comment by frntc** [comment]
Yes, this is exactly what you need to do (cut and close the other side)! Good to hear that finally someone needs that solder pad :-)
